bin/sl-pmctl.js

```diff
@@ -435,91 +432,65 @@ function extra() {
 
 function remoteRequest(pmctl, cmd, callback) {
   var endpoint = url.parse(pmctl);
-
   if (!endpoint.protocol) {
-    return client.request(pmctl, cmd, callback);
+    pmctl = 'http+unix://' + path.resolve(pmctl);
+    remoteHttpRequest(pmctl, cmd, callback);
```

Comment:
My PR contains dd96b12, which applies a further refactoring to this. This suggestion is optional, but it would reduce code churn if you squashed that commit into this one.